backend/app.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express')
+const path = require('path');
+const logger = require('morgan');
+const bodyParser = require('body-parser');
+
+const app = express();
+
+// Set-up View Engine for displaying data
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'ejs');
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(express.static(path.join(__dirname, 'public')));
+
+// example query from neo4j/sandbox that looks for specific DataCenters
+query = 'MATCH network = (dc:DataCenter {name:"DC1",location:"Iceland, Rekjavik"}) -[:CONTAINS]->(:Router) -[:ROUTES]->(:Interface) RETURN network'
```

Comment:
Querying for dataCenters in `Iceland, Rekjavik` to output their location (`Iceland, Rekjavik`) isn't the best exmample. So I suggest adjusting the query and output below.

Instead of returning the whole network return some values only like:
```suggestion
query = 'MATCH network = (dc:DataCenter {name:"DC1",location:"Iceland, Rekjavik"}) -[:CONTAINS]->(r:Router) -[:ROUTES]->(i:Interface) RETURN r.name as name, i.ip as ip'
```